### ISSUE TITLE
Show server information like privacy policy, file upload limits, etc.

### ISSUE THREAD

**Issue by pafcu** [vector-im/element-web#5193]
It would be nice if Riot could show information about the privacy policy of the user's homeserver (e.g. when signing up), file upload limits, and other capabilities of the chosen homeserver.

**Comment by XDASyndicate** [comment]
I don't think you can set privacy policies and those settings on the homeserver?

**Comment by t3chguy** [comment]
If you still value this enhancement I suggest opening an issue in https://github.com/matrix-org/matrix-spec

**Comment by t3chguy** [comment]
Actually my bad, for the datums requested they are already available on the API, it is just unclear in what non-technical manner they would be presented
